src/ServiceControl.Audit/Infrastructure/SetupBootstrapper.cs

```diff
@@ -72,6 +73,7 @@ public async Task Run(string username)
             var documentStore = new EmbeddableDocumentStore();
             containerBuilder.RegisterInstance(documentStore).As<IDocumentStore>().ExternallyOwned();
             containerBuilder.RegisterInstance(settings).SingleInstance();
+            containerBuilder.RegisterType<MigrateKnownEndpoints>();
```

Comment:
Was this necessary for it to work? I thought installer are automatically picked up by scanning and registered by NServiceBus. Just curious not suggestion you have to change it

Reply:
Error when creating the instance without doing this.